pint/testsuite/pandas_test_interface.py

```diff
@@ -146,7 +147,35 @@ class TestInterface(base.BaseInterfaceTests):
     pass
 
 class TestMethods(base.BaseMethodsTests):
-    pass
+
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
+    # See test_setitem_mask_broadcast note
+    @pytest.mark.parametrize('dropna', [True, False])
+    def test_value_counts(self, all_data, dropna):
+        all_data = all_data[:10]
+        if dropna:
+            other = all_data[~all_data.isna()]
+        else:
+            other = all_data
+
+        result = pd.Series(all_data).value_counts(dropna=dropna).sort_index()
+        expected = pd.Series(other).value_counts(
+            dropna=dropna).sort_index()
+
+        self.assert_series_equal(result, expected)
+    
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```

Comment:
I'd like to keep these warnings popping up unless there's a good reason to silence them?